report/src/bluetooth_presence_methods/our_approach.tex

```diff
@@ -0,0 +1,59 @@
+\section{Our Approach}\label{sec:our_approach}
+Based on our prior discussion in Section \ref{sec:ChoiceOfTechnique} on potential approaches, we have opted to use a fingerprinting technique for our system.
+Our methodology is split into two main phases: data collection and classification.
+Each phase plays a critical role in successfully implementing this technique and, when combined, they offer an effective method of presence detection.
```

Comment:
I feel like this paragraph is concluding on something we havent tested yet at this stage in the report. I would exclude it. 
Maybe instead give a brief explanation of the two phases, or perhaps write what we can expect to read in the coming sections.